packages/components/organisms/f-checkout/src/exceptions/exceptions.js

```diff
@@ -22,12 +23,12 @@ class UpdateCheckoutError extends Error {
 }
 
 class PlaceOrderError extends Error {
-    constructor (message, shouldShowInDialog = false) {
+    constructor (message, errorCode) {
         super(message);
         this.messageKey = 'errorMessages.genericServerError';
         this.eventToEmit = EventNames.CheckoutPlaceOrderFailure;
         this.logMessage = 'Place Order Failure';
-        this.shouldShowInDialog = shouldShowInDialog;
+        this.shouldShowInDialog = checkoutIssues[errorCode];
```

Comment:
If I read the code correctly, this would make `this.shouldShowInDialog` an object, but we want the value for the `shouldShowInDialog`  key from that object in here:
```suggestion
        const issue = checkoutIssues[errorCode] || {}; // in case the errorCode is unrecognised E.g. from a later API version
        this.shouldShowInDialog = issue.shouldShowInDialog || false; // or `true` if that makes more sense as a default value here
```